application/event_processor/reporting.py

```diff
@@ -89,17 +91,25 @@ def log_invalid_nhsuk_pharmacy_postcode(nhs_entity: NHSEntity, dos_service: DoSS
             "dos_service": dos_service.uid,
         },
     )
+    metrics.set_namespace("UEC-DOS-INT")
+    metrics.set_property("level", "WARNING")
+    metrics.set_property("message", error_msg)
```

Comment:
can we set some of the other props such as correlation_id so that the logs rows are populated and also so these come back in a correlation id search in the log insights